spec/datasets/djornl/edge_type.yaml

```diff
@@ -52,3 +52,20 @@ oneOf:
     title: ATRM TF to Target LitCurated 01082020 TranscriptionFactorToGene
     description: Contains literature mined and manually curated TF regulatory interactions for A.thaliana from 1701 TFFs from PlantTFDB 2.0 and 4663 TF-associated interactions. These were manually filtered (e.g. FPs, PPI interactions removed). They then added some from other sources. Downloaded from http://atrm.cbi.pku.edu.cn/download.php
 
+  - const       : 'Metabolic_AraCyc_Gene_to_Gene'
+    title       : 'AraCyc_Gene_to_Gene'
+    description : "AraCyc.  GeneA connects to GeneB if they are both enzymes and are linked by a common substrate or product. E.g. RXNA (GeneA) → Compound1 → RXNB (GeneB). Here GeneA connects to GeneB due to Compound1.  Date accessed 03/18/2021.  All enzymatic reactions and associated genes (enzymes) and compounds were downloaded from AraCyc as smarttables, and transformed by custom R script to generate the appropriate network format linking gene to gene based on intermediary compounds.  Bringing to-spec, 322 non-standard edges were removed and 3,243 duplicates were removed."
```

Comment:
These values don't need to have `"` or `'` around them.

The `title` values should be plain English; no need for underscores separating the words (unless you genuinely want them there!).

Can you also take a look at the way that the other networks have been named and see if you can use similar nomenclature for these networks?

Small but picky thing - can you add a line break between each block to make it easier to read?